src/napari_imagej/widgets/menu.py

```diff
@@ -5,6 +5,7 @@
 from pathlib import Path
 from typing import Any, Dict, Optional, Union
 
+from imagej.images import is_arraylike
```

Comment:
The `is_arraylike` function is also available in `scyjava` directly now.

Reply:
@ctrueden yeah, I see it in scyjava 1.8.1.

Looks like it's in the `_arrays` module, though. Is it intended to be used downstream?

Reply:
It is [imported as a public function](https://github.com/scijava/scyjava/blob/1.8.1/src/scyjava/__init__.py#L73-L77).

Reply:
Oh, I missed that. Thanks!

Reply:
Fixed in 3c1772e